Task08.cpp

```diff
@@ -0,0 +1,25 @@
+
+#include <iostream>
+using namespace std;
+int pow(int num, unsigned N) {
+
+    int result = 0;
+
+    if (N == 0) {
+        result = 1;
+    }
+    else if (N > 0) {
+        for (N; N >= 1; N--) {
+            result *= num;
+        }
+    }
+    return result;

```

Comment:
A more clear approach of writing this code
if(n == 0) return 1
if(N < 0) return 0
// then you do the for loop with the result